.github/workflows/test_e2e.yml

```
@@ -0,0 +1,64 @@
+# See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: 'Test: E2E'
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/test_e2e.yml'
+      - 'services/.env.example'
+      - 'services/121-service/**'
+      - 'services/mock-service/**'
+      - 'interfaces/Portal/**'
+      - '!**.md'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+
+      - name: Set ENV-variables for test-environment
+        run: |
+          cp ./services/.env.example services/.env
+
+      - name: Run Services with Docker
+        working-directory: ./services
+        run: |
+          docker --log-level 'warn' compose -f docker-compose.yml up -d --quiet-pull
+
+      - name: Start Portal
+        working-directory: ./interfaces/Portal
+        run: |
+          npm install
+          npm start -- --port 8888 > /dev/null 2>&1 &
```

I'd say that using a "production mode"-version of the Portal gives more realistic results;
So using:

```suggestion
          npm run start:debug-production > /dev/null 2>&1 &
```

This will then run on port `8088`, so this would need to be adjusted below in the health-check.